static/js/types/notification.d.ts

```diff
@@ -7,11 +7,18 @@ export interface Notification {
   type: ValueOf<typeof NotificationSeverity>;
 }
 
+export interface QueuedNotification {
+  state: {
```

Comment:
`state` can be null. Please change this line to:
```
  state?: {
```
This way, we can prevent this error:
```
Uncaught TypeError: Cannot read properties of null (reading 'queuedNotification')
    at useNotification (useNotification.tsx:13:19)
    at InstanceList (InstanceList.tsx:34:33)
    [...]
```

Reply:
right, I missed that.